manifests/init.pp

```diff
@@ -40,30 +40,68 @@
 #   *If* the $servers array above starts with '127.0.0.1' or '::1', then
 #   the system will set itself up as a caching nameserver unless this is set
 #   to false.
+# @param manage_via_nmcli
```

Comment:
@kendall-moore  Can we change this to `use_nmcli` to match the majority of the other modules? Not critical, just nice to have.

Reply:
Sure, didn't see this but let me change it now and push.